_includes/tutorials/aggregating-average/kstreams/code/docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+---
+version: '3.5'
```

Comment:
Can this be version 2 like the other tutorials?

Reply:
no, networks section isn't supported in ver 2